Arm/joint.hpp

```diff
@@ -94,6 +94,37 @@ class Joint
     return int(position_);
   }
 
+
+// edits the min angle. Useful for specific robot modes where full traversal might not be a good idea.
+  void SetMinAngle(float newmin)
```

Comment:
```suggestion
  void SetMinAngle(float new_min)
```
Google style guide naming convention